pkgs/top-level/all-packages.nix

```diff
@@ -9513,6 +9513,12 @@ with pkgs;
     installLocales = config.glibc.locales or false;
   };
 
+  # Provided by libc on Operating Systems that use the Extensible Linker Format.
+  elf-header =
```

Comment:
Have you seen if ```libelf``` above would work? Not sure how that works currently but it might be a little bit more generic of the name. elf-header IMO is a little two awkward. No package will ever list a dependency on "elf-header" but may for libelf.

Reply:
`libelf` is something else. I bet paxctl could be *re*written against it, but it's not a drop-in replacement. (`libelf` also looks more like the darwin DTrace header.)

Reply:
Hmm... I guess it's okay then. It's just semantically kind of ugly to me.